Source/WebCore/Modules/fetch/FetchRequestInit.idl

```diff
@@ -41,5 +41,6 @@ dictionary FetchRequestInit {
     boolean keepalive;
     // FIXME: 'signal' should be of type AbortSignal?.
     any signal;
+    RequestPriority priority;
```

Comment:
Can we add EnabledBySetting=PriorityHintsEnabled?

Reply:
Done.